README.md

```diff
@@ -38,7 +38,7 @@ Some companies don't publish adequate contact information on their website. In t
 
 ### Addresses
 
-We want the addresses to formatted consistently, so don't just copy the information directy from the company's website. Try to format the address according to the standards of the respective country. For many countries, we already have records with examples on how to format the addresses. Otherwise, the Universial Postal Union has [resources](http://www.upu.int/en/activities/addressing/postal-addressing-systems-in-member-countries.html) on how to properly format addresses for many countries. In addition, there is a very [comprehensive guide](https://www.bitboost.com/ref/international-address-formats.html) offered by BitBoost Systems.
+We want the addresses to formatted consistently, so don't just copy the information directy from the company's website. Try to format the address according to the standards of the respective country. For many countries, we already have records with examples on how to format the addresses. Otherwise, the Universal Postal Union has [resources](http://www.upu.int/en/activities/addressing/postal-addressing-systems-in-member-countries.html) on how to properly format addresses for many countries. In addition, there is a very [comprehensive guide](https://www.bitboost.com/ref/international-address-formats.html) offered by BitBoost Systems.
```

Comment:
While you're at it: This should be "We want the addresses to **be** formatted consistently". ;)

Reply:
Yes, sure! Will do. Thanks.